src/ChainSafe.Gaming/MultiCall/IMultiCallConfig.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace ChainSafe.Gaming.MultiCall
+{
+    public interface IMultiCallConfig
```

Comment:
is this useful? it's not referenced anywhere and we use the concrete config class for DI

Reply:
@robGG1997 The config interface is needed for the dependency injection but the config could later on be expanded to additional features, I left having the option of a config object so that the work flow is available down the line

Reply:
I believe DI uses the concrete class not the interface, however I see your point we could expand on it in the future and it doesn't hurt to keep it